src/Workspaces/Core/Portable/Formatting/Formatter.cs

```diff
@@ -360,5 +361,8 @@ private static IEnumerable<TextSpan> AggregateSpans(IEnumerable<TextSpan> spans)
 
             return aggregateSpans;
         }
+
+        public static Task<Document> OrganizeImportsAsync(Document document, CancellationToken cancellationToken)
+            => document.GetLanguageService<IOrganizeImportsService>().OrganizeImportsAsync(document, cancellationToken);
```

Comment:
should we do nothing for langauges that do not support this service?

Reply:
1. we should add docs to this method now that it's public.  
2. CancellationToken should be optional on this public method.  
3. consider either throwing or performing a no-op for languages that do not export an IOrganizeImportsService

Reply:
> CancellationToken should be optional on this public method

I would strongly support not making the parameter optional. I believe the remaining cases are all mistakes that only continue to exist for compatibility reasons.

Reply:
This would go against the roslyn API design pattern we have for (i believe) 100% of all other cancellable APIs.  I don't see any reason to go against the pattern unless the team as a whole decides that we're ditching the old pattern and goin with non-optional cancellation from now on.